### ISSUE TITLE
Nextflow output

### ISSUE THREAD

**Issue by Mustardburger** [IllustratedMan-code/nextflow_assignment#1]
I'm trying to merge two fa files, but I'm having this issue: Missing output file(s) `ctrl_rep_C` expected by process `merge_reads (3)`. My .nf file is as follows:
```nextflow
#!/usr/bin/env nextflow

params.folder = "$baseDir/seqs/*read{1,2}.fa"
read_pair_channels = Channel.fromFilePairs(params.folder)

process merge_reads {

    input:
    tuple val(read_id),path(read_file)

    output:
    path(read_id)

    script:
    """
    cat > "$read_id".fa
    cat $read_file >> "$read_id".fa
    """
}

workflow {
    merge_reads(read_pair_channels)
}
```

**Comment by Mustardburger** [comment]
This is for task 1. So I was able to fix the bug, but there're new problems with my code below. First, when I run the code, the merge_read folder is created, but there's nothing in it. Second, when I check the corresponding folder in work, I can see only 1 .fa file, instead of multiple .fa files. That only .fa file has a correct format though: its name is the shortname, and inside it concatenates the read1 and read2 .fa files
```nextflow
#!/usr/bin/env nextflow

params.folder = "$baseDir/seqs/*read{1,2}.fa"
read_pair_channels = Channel.fromFilePairs(params.folder)

process merge_reads {

    publishDir "merge_reads/"

    input:
    tuple val(read_id),path(read_file)

    output:
    path "${read_id}.fa"

    script:
    """
    touch "${read_id}.fa"
    cat $read_file >> "${read_id}.fa"
    """
}

workflow {
    merge_reads(read_pair_channels)
}
```

**Comment by IllustratedMan-code** [comment]
A tip: use triple backticks to specify code blocks
``````
```nextflow
```
``````
I went ahead and edited your comments to use them.

**Comment by IllustratedMan-code** [comment]
Do you mind forking the repo and pushing your changes so that I can see your seq directory?

**Comment by Mustardburger** [comment]
Hmm I'll try once I get back home

**Comment by IllustratedMan-code** [comment]
Looks like you figured it out, merged your changes in #2
